.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-- 2.7
 - 3.5
```

Comment:
I can but it would fail because edx-celeryutils does not support 3.8 (It has a dependency on an old version of celery that doesn't support 3.6 or above)

Reply:
OK to leave it out hen.